src/Tribe/Date_Utils.php

```diff
@@ -1167,6 +1167,34 @@ public static function unescape_date_format( $date_format ) {
 			// Why so simple? Let's handle other cases as those come up. We have tests in place!
 			return str_replace( '\\\\', '\\', $date_format );
 		}
+
+		/**
+		 * Gets the timezone string associated with an event.
+		 *
+		 * Will fall back to WP timezone string and to system timezone string in this order.
+		 *
+		 * @since TBD
+		 *
+		 * @param $event_id
+		 *
+		 * @return mixed|string|void
+		 */
+		public static function get_event_timezone_string( $event_id ) {
+			$event_timezone_string = get_post_meta( $event_id, '_EventTimezone', true );
+
+			if ( empty( $event_timezone_string ) ) {
+				$event_timezone_string = get_option( 'timezone_string', false );
+				if ( empty( $event_timezone_string ) ) {
+					$event_timezone_string = date_default_timezone_get();
```

Comment:
non-blocker for sure, just commenting: would love to see this part reduced to one line:

`return date_default_timezone_get();`